jfrog-cli/artifactory/commands/npm/publish.go

```diff
@@ -161,9 +161,9 @@ func (npmp *npmPublish) doDeploy(target string, artDetails *config.ArtifactoryDe
 
 func (npmp *npmPublish) saveArtifactData() error {
 	log.Debug("Saving npm package artifact build info data.")
-	var buildArtifacts []buildinfo.Artifact
+	var buildArtifacts []buildinfo.InternalArtifact
```

Comment:
Wasn't too sure if this would break something; this change implies that the `build.name` and `build.number` properties will be set on these artifacts at the moment the build is uploaded.

Can someone shed their light on whether or not this has nasty side-effects?